### ISSUE TITLE
Scrape action should not be performed during startup when scan interval set to 0

### ISSUE THREAD

**Issue by Paul-Vdp** [danieldotnl/ha-multiscrape#359]
Title says it all.
My scrape sensor relies on 'params' for some parts of the resource. Because these params have not (yet) been initialized on startup, the (unwanted and unneeded !) scrape results in errors.

**Comment by danieldotnl** [comment]
What should the value of the sensors be on startup in this case?

**Comment by Paul-Vdp** [comment]
To be honest : don't care, because I don't need nor use them at that moment - that's why their scan interval is set to 0 in the first place.
Or maybe a more reasonable and acceptable answer would be : restored from their previous value, as with most other sensors ?
As the running of the scrape on startup has got nothing to do with any need of these sensors to be refreshed/updated ...

**Comment by danieldotnl** [comment]
I agree with @Paul-Vdp and I will work on implementing this. It's not a small feature request though, so it will take some time.

**Comment by Paul-Vdp** [comment]
Much obliged @danieldotnl 
I realize it is not a simple change, but I am confident you will manage ;-)

**Comment by Paul-Vdp** [comment]
@danieldotnl Any progress made on this ? Just asking ;-)

**Comment by saulleighton23** [comment]
scan_interval: 0 will be a really useful feature when implemented - I wholly support it.

**Comment by danieldotnl** [comment]
Please try pre-release [v8.0.0](https://github.com/danieldotnl/ha-multiscrape/releases/tag/v8.0.0)!

**Comment by Paul-Vdp** [comment]
Really appreciate you took care of my (and of others) two requests.
But I am truly sorry to say that at least the problem with the unwanted scrape on startup does not seem to be resolved, as it still happens to me ... ?

**Comment by danieldotnl** [comment]
No worries, that's why it is a pre-release 😊
Can you share your config and debug logging?

**Comment by Paul-Vdp** [comment]
Sure, here you go :
`multiscrape:
  - name: Scrape Flights
    resource: "https://www.tuifly.be/flight/nl/search?adults=2&nearByAirports=false&isOneWay=true"
    params:
      flyingFrom%5B%5D: "{{ states('var.flyfrom')|default('BRU') }}"
      flyingTo%5B%5D: "{{ states('var.flyto')|default('HRG') }}"
      depDate: "{{ states('input_datetime.scrape_date')|default(now().date()) }}"
    headers:
      User-Agent: "Mozilla/5.0 (iPad; CPU OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148"
#    log_response: true
    scan_interval: 0
    sensor:
      - unique_id: scrape_flights
        name: Scrape Flights
        value_template: "Flights"
        attributes:
          - name: flights
            value_template: >
              {% set ns = namespace(flights=[]) %}
              {% set split = value.split('"flightViewData":')[1].split(',"seasonEndDate"')[0]|from_json %}
              {% for flight in split %}
                {% set weekDay = flight.departureDate|as_datetime|as_timestamp|timestamp_custom('%a') %}
                {% set detail = '{"weekDay":"' + weekDay + '", "departureDate":"' + flight.departureDate + '", "adultPrice":"' + flight.adultPrice + '", "depTime":"' + flight.journeySummary.depTime +'", "arrivalTime":"' + flight.journeySummary.arrivalTime + '", "journeyDuration":"' + flight.journeySummary.journeyDuration + '", "journeyType":"' + flight.journeySummary.journeyType + '", "arrivalAirport":"' + flight.flightsectors[0].arrivalAirport.name + '"}' %}
                {% set ns.flights = ns.flights + [detail] %}
              {% endfor %}
              {% set newflight = ns.flights|replace('\'', '') %}
              {% set oldflights = state_attr('sensor.scrape_flights', 'flights') %}
              {% if states('var.scrape_counter')|int == 1 or oldflights == [] or oldflights == null %}
                {{ newflight }}
              {% else %}
                {% set flights = oldflights + newflight|from_json %}
                {{ flights }}
              {% endif %}
            on_error:
              value: last
`
After startup, I get in the logs :
`This error originated from a custom integration.

Logger: custom_components.multiscrape.entity
Source: custom_components/multiscrape/entity.py:158
integration: Multiscrape (documentation, issues)
First occurred: 11:29:46 (1 occurrences)
Last logged: 11:29:46

Scrape Flights # Scrape Flights # flights # Unable to extract data from HTML`
which to me shows that it is still running the scrape on startup, no ?

**Comment by danieldotnl** [comment]
Fixed in [v8.0.1](https://github.com/danieldotnl/ha-multiscrape/releases/tag/v8.0.1)!
